### PR TITLE
QAE-823 - E2E test case for footer menu hide on devices

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Footer builder → Footer menu → Hide on devices

### Screenshots

https://d.pr/i/EnGMLq

### How has this been tested?

npm run test:e2e:interactive tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+				},
+			},
+		};
+		await setCustomize( hideOnTablet );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await setBrowserViewport( 'medium' );
+		await scrollToElement( '#colophon' );
+		await page.waitForSelector( '.footer-widget-area[data-section="section-footer-menu"]' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js:50]:
```suggestion
		await page.waitForSelector( '.ast-header-break-point .footer-widget-area[data-section="section-footer-menu"]' );
```

File: tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+				},
+			},
+		};
+		await setCustomize( hideOnMobile );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await setBrowserViewport( 'small' );
+		await scrollToElement( '#colophon' );
+		await page.waitForSelector( '.footer-widget-area[data-section="section-footer-menu"]' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js:75]:
```suggestion
		await page.waitForSelector( '.ast-header-break-point .footer-widget-area[data-section="section-footer-menu"]' );
```

File: tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js (hunk 1 above)
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js:50]: Screenshot:- https://share.bsf.io/d5u97mmw

File: tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js]: Resolved

**Review by GauravKhupse — APPROVED** [review]

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/elements/footer-menu/hide-on-devices.test.js]: resolved

**Pull request merged**